packages/alert/src/Alert.tsx

```diff
@@ -50,6 +58,26 @@ export interface AlertProps extends React.HTMLAttributes<HTMLElement> {
    * The variant to use. Defaults to "relative".
    */
   type?: 'relative' | 'toast' | 'banner'
+
+  /**
+   * Show / Hide close button
+   */
+  showCloseButton?: boolean
```

Comment:
Similar to https://github.com/little-spoon-dev/design-system/pull/655/files#r988505762. If we are going to have a second prop to control this, we should ideally make sure that this can only be true if `onClose` is provided. 

This case might be trickier but worth giving it a try. It should be something like this:

``
type Props = {
  showCloseButton: true,
  onClose: () => void
} | {
  onClose?: () => void
}

Reply:
I think `onClose` should be a required field. what do you think?

Reply:
I think not necessarily. In some cases, you don't want the user to have the option to close it. To me, it should be required if it's type `toats` or you have `showCloseButton`.